design/pool-configuration.md

```diff
@@ -0,0 +1,385 @@
+# Address Pool configuration options
```

Comment:
Just a general question for maintainers. Do we have a naming convention for design proposals?

Reply:
AFAIK, no. But maybe we can move ones already implemented to a subdir, assume most will be implemented so this will still be manageable?